lib/phoenix/template.ex

```diff
@@ -126,9 +145,11 @@ defmodule Phoenix.Template do
   @doc false
   defmacro __using__(options) do
     root = Dict.fetch! options, :root
+    pattern = Dict.get options, :pattern, "*"
 
     quote do
       @template_root Path.relative_to_cwd(unquote(root))
+      @lookup_pattern unquote(pattern)
```

Comment:
Glob is when you include the asterisk. You don't need to have an asterisk in your pattern. You could use "{foo,bar,baz}.html" if you want to select some templates, for example.